CharlieBackend.Business/Services/AccountService.cs

```diff
@@ -105,18 +105,32 @@ public async Task<Account> GetAccountCredentialsByIdAsync(long id)
             return null;
         }
 
-        public async Task<IList<AccountDto>> GetAllNotAssignedAccountsAsync()
+        public async Task<Result<IList<AccountDto>>> GetAllNotAssignedAccountsAsync()
```

Comment:
there's no point in using Result for list methods (since there's no validation happens)